engine/api/src/main/scala/pl/touk/nussknacker/engine/api/CustomStreamTransformer.scala

```diff
@@ -26,6 +26,8 @@ abstract class CustomStreamTransformer {
   // TODO: remove after full switch to ContextTransformation API
   def canHaveManyInputs: Boolean = false
 
+  def canBeEnding: Boolean = false 
```

Comment:
maybe we should have it in ContextTransformation API instead of here?

Reply:
For me both options are ok. This information can be easily accessible both from `AbstractContextTransformation`/`AbstractContextTransformationDef` and `CustomStreamTransformer`/`CustomTransformerAdditionalData`.

I think that is a matter whether we want to support ending custom nodes only in brand new ContextTransformation API or maybe also in other projects where developers returns their own types to interpret by custom runtimes (without forcing to wrap with ContextTransformation).